src/plugins/ui_core_views/evaluation_data_validation.ts

```diff
@@ -86,6 +87,10 @@ export class EvaluationDataValidationPlugin extends UIPlugin {
     return validationResult.isValid ? undefined : validationResult.error;
   }
 
+  isValidFormula(value: string): boolean {
```

Comment:
can be moved with other private methods down below
```suggestion
  private isValidFormula(value: string): boolean {
```